go/cmd/vtctldclient/cli/shards.go

```diff
@@ -41,3 +46,78 @@ func ParseKeyspaceShards(args []string) ([]*vtctldatapb.Shard, error) {
 
 	return shards, nil
 }
+
+// ReplicatingTablet is a struct to group a Tablet together with its replication
+// Status.
+type ReplicatingTablet struct {
+	Status *replicationdatapb.Status
+	Tablet *topodatapb.Tablet
+}
```

Comment:
Did you consider writing it like this?
```
type ReplicatingTablet struct {
	topodatapb.Tablet
	Status *replicationdatapb.Status
}
```

Reply:
I did, and decided not to for a silly reason. Basically I was worried "what if Tablet and Status each have a field with the same name, you wouldn't be able to tell the difference" but now I remember that Go will just force you to disambiguate so it's actually not a problem at all.

I'll make this change!